aspnet/fundamentals/servers.rst

```diff
@@ -13,39 +13,44 @@ ASP.NET Core is completely decoupled from the web server environment that hosts
 
 `View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/servers/sample>`__
 
-Servers and commands
---------------------
+Servers and WebHostBuilderExtensions
+------------------------------------
 
-ASP.NET Core was designed to decouple web applications from the underlying HTTP server. Traditionally, ASP.NET apps have been windows-only hosted on Internet Information Server (IIS). The recommended way to run ASP.NET Core applications on Windows is using IIS as a reverse-proxy server. The HttpPlatformHandler module in IIS manages and proxies requests to an HTTP server hosted out-of-process. ASP.NET Core ships with two different HTTP servers:
+ASP.NET Core was designed to decouple web applications from the underlying HTTP server. Traditionally, ASP.NET apps have been windows-only hosted on Internet Information Server (IIS). The recommended way to run ASP.NET Core applications on Windows is using IIS as a reverse-proxy server. The ASP.NET Core Module in IIS manages and proxies requests to the Kestrel HTTP server hosted out-of-process. ASP.NET Core ships with two different HTTP servers:
```

Comment:
"The recommended way to run ASP.NET Core applications on Windows is still using IIS, but as a reverse-proxy server."